database.json

```diff
@@ -669,6 +669,12 @@
     "repo": "deno-structured-logging",
     "desc": "A structured logging module for Deno"
   },
+  "deno_string_validator": {
+    "type": "github",
+    "owner": "okwudav",
+    "repo": "deno-string-validator",
+    "desc": "String validation for Deno, inspired by Validator.js => https://github.com/validatorjs/validator.js"
+  },
```

Comment:
the database entries should be sorted alphabetically. Please insert your new entry in the correct location.

in this case before deno_structured_logging and after deno_search